selfdrive/manager/process.py

```diff
@@ -63,12 +63,19 @@ def join_process(process: Process, timeout: float) -> None:
     time.sleep(0.001)
 
 
+def enabled_callback(params: Params, CP: car.CarParams) -> bool:
+  return True
+
+
+def disabled_callback(params: Params, CP: car.CarParams) -> bool:
+  return False
+
+
 class ManagerProcess(ABC):
   daemon = False
   sigkill = False
-  onroad = True
-  offroad = False
-  callback: Optional[Callable[[bool, Params, car.CarParams], bool]] = None
+  onroad_callback: Callable[[Params, car.CarParams], bool] = enabled_callback
+  offroad_callback: Callable[[Params, car.CarParams], bool] = disabled_callback
```

Comment:
do we really need two different callbacks?

Reply:
If we have one callback, then you need to always make sure to implement `started` which I thought we wanted to refactor out

Reply:
I haven't found a cleaner way, since we use the callback as an OR to allow processes that normally don't run offroad to run (like driverview), but in the case of the body, we wanted it to be an AND while onroad.

If there's an onroad and offroad callback, we can clearly see the logic in the process_config.py file. Let me know if you have any other ideas.